notebooks/fixtures/config_test_file_a.nml

```diff
@@ -0,0 +1,4 @@
+&values
+  greeting = "Hello"
+  recipient = "World"
+/
```

Comment:
```suggestion
/
```
Here and below, adding suggestions to add a final newline to files.

Reply:
Updated file to show textually different but semantically identical example you provided. Same for below.